.travis.yml

```diff
@@ -45,7 +48,7 @@ before_install:
   - sudo chown -R elasticsearch:elasticsearch /etc/default/elasticsearch
   - sudo systemctl start elasticsearch
 install:
-  - bundle install --path vendor/bundle
+  - bundle install --jobs=3 --retry=3 --path vendor/bundle
```

Comment:
We are running 2 VMs so why does `jobs` here equal 3? I would have expected 2.

+1 for retry!!!

Reply:
This runs inside each VM and configures how many parallel downloads/installs Bundler performs:

```
--jobs=[<number>], -j[<number>]
  The maximum number of parallel download and install jobs. The default is 1.
```

The 3 came straight from the Travis Ruby docks, I didn't think about it too much tbh. Our Bundle is cached anyway, so hopefully, we don't need too many downloads at any given time anyway.

Reply:
OH, thank you for the explanation! I thought it was one for each VM 🤦‍♀  whoops!